docs/customizing_page_views.md

```diff
@@ -2,12 +2,22 @@
 title: Customizing page views
 ---
 
-In order to change the appearance of any page,
-you can write custom Rails views.
+You can provide replacements for any of Administrate's templates.
+This way you can change the appearance of any page or element of
+the interface.
+
+In general, you can override any of the views under Administrate's
+[/app/views](https://github.com/thoughtbot/administrate/tree/master/app/views).
+For example, say that you want to customize the template used for flash
+messages. You can provide your own as
+`/app/views/administrate/application/_flashes.html.erb`, and it will replace
+Administrate's own.
+
+Figuring out which views are available and where can be repetitive. You can
+spare yourself some effort by using the built-in view generators.
```

Comment:
What do you think about using a Markdown reference link in this section to make it a bit easier to read?

```suggestion
[/app/views][1].
 For example, say that you want to customize the template used for flash
 messages. You can provide your own as
 `/app/views/administrate/application/_flashes.html.erb`, and it will replace
 Administrate's own.

 Figuring out which views are available and where can be repetitive. You can
 spare yourself some effort by using the built-in view generators.
 
 [1]: https://github.com/thoughtbot/administrate/tree/master/app/views
```

Reply:
Not sure if the specific suggestion (as in: the diff) that you propose is the one you intended? The idea is good though. I just pushed a change. Let me know whether I understood correctly or not.

Reply:
Ah yeah, sorry the diff ended up being really hard to understand. That is what I meant!